src/apps/schema/src/appRevamp/components/configs.ts

```diff
@@ -375,7 +375,28 @@ const FORM_CONFIG: { [key: string]: InputField[] } = {
   date: [...COMMON_FIELDS],
   datetime: [...COMMON_FIELDS],
   dropdown: [],
-  images: [],
+  images: [
+    ...COMMON_FIELDS,
+    {
```

Comment:
Adding this here would render the checkboxes on the "Details" tab as well. 

But since @agalin920 suggested that these should be on the details tab anyway, then I suggest making a custom component on `FieldFormInput.tsx` to handle this type since it's going to show a dropdown on `checked === true`.

Reply:
Gotcha